inc/opendefs.h

```diff
@@ -293,6 +303,24 @@ typedef struct {
    uint8_t       l2_joinPriority;                // the join priority received in EB
    bool          l2_IEListPresent;               //did have IE field?
    bool          l2_joinPriorityPresent;
+   //START OF TELEMATICS CODE
+   //security
+   bool			 	 l2_security;				 //flag for security enabled/disabled

```

Comment:
please respect indendation and alignment.
Also, please configure your editor to not include any tabs. We don't want a single tab anywhere. tabs are evil.